packages/openapi-generator/src/wrapper-files/api-file.ts

```diff
@@ -106,3 +109,9 @@ function getRequestBodyParams(operation: OpenApiOperation): Parameter[] {
       ]
     : [];
 }
+
+function getType(parameter: Parameter) {
+  return parameter.enum
+    ? parameter.enum.map(e => `'${e}'`).join(' | ')
```

Comment:
I guess this is only correct for strings, but not for integers and booleans (and maybe others). Consider all possible types.